src/js/feedback/actions/index.js

```diff
@@ -0,0 +1,62 @@
+import { apiRequest } from '../../common/helpers/api';
+import { isValidEmail } from '../../common/utils/validations';
+
+export const REVEAL_FORM = 'REVEAL_FORM';
+export const SET_FORM_VALUES = 'SET_FORM_VALUES';
+export const SEND_FEEDBACK = 'SEND_FEEDBACK';
+export const FEEDBACK_RECEIVED = 'FEEDBACK_RECEIVED';
+export const FEEDBACK_ERROR = 'FEEDBACK_ERROR';
+export const CLEAR_FEEDBACK_ERROR = 'CLEAR_FEEDBACK_ERROR';
+
+const errorMessage = 'An error occurred while trying to submit the form. We apologize for the inconvenience.';
+
+function runValidation({ description, email, shouldSendResponse }) {
+  const formErrors = {};
+  if (description !== undefined) {
+    formErrors.description = description.length > 0 ? null : 'Please enter a description';
+  }
+  if (shouldSendResponse === false) {
+    formErrors.email = null;
+  } else if (email !== undefined) {
+    formErrors.email = isValidEmail(email) ? null : 'Please enter a valid email';
+  }
+  return formErrors;
+}
+
+export function revealForm() {
+  return { type: REVEAL_FORM };
+}
+
+export function setFormValues(formValues) {
+  return {
+    type: SET_FORM_VALUES,
+    formValues,
+    formErrors: runValidation(formValues)
+  };
+}
+
+export function sendFeedback() {
+  return (dispatch, getState) => {
+    const { formValues } = getState().feedback;
```

Comment:
`getState` is sometimes necessary, but it's also very easy to break without knowing it if you change the structure of the Redux state. You could also pass formValues as a parameter.